### ISSUE TITLE
This repo is too large for Bower

### ISSUE THREAD

**Issue by jiyinyiyong** [ejci/favico.js#48]
It's 4.8M large when I try to install from Bower, that's really large. Considering the bad networking in China it would take some minutes to download. Could you figure out a solution to reduce the size...

BTW I noted there is samller one here but outdated: https://github.com/fastdivision/favico.js-lite

**Comment by adam-lynch** [comment]
Isn't there a bower equivelant of .npmignore? I know there are `main`

**Comment by adam-lynch** [comment]
Isn't there a bower equivelant of `.npmignore`? I know there can be an `ignore` property in your `bower.json` but not sure if it actually prevents them from being downloaded.

**Comment by aitorllj93** [comment]
:+1:  @adam-lynch bower's ignore just does exactly that 

Copy paste from http://bower.io/docs/creating-packages/#bowerjson-specification:

ignore

Array of Strings

Recommended

A list of files for Bower to ignore when installing your package.

Note: symbolic links will always be ignored. However bower.json will never be ignored.

The ignore rules follow the same rules specified in the gitignore pattern spec.

**Comment by aitorllj93** [comment]
I have made a bower install (I saw that the bower.json already contains the ignore property) and the ignored files aren't in the installation folder.
 But it seems that bower removes them from the folder after downloading it :(

```bash
bower favico.js#*           not-cached git://github.com/ejci/favico.js.git#*
bower favico.js#*              resolve git://github.com/ejci/favico.js.git#*
bower favico.js#*             download https://github.com/ejci/favico.js/archive/0.3.9.tar.gz
```

That archive (even being compressed)  has the 4,8M lenght as mentioned by @jiyinyiyong 

I think is most an issue of how Bower manages the packages download (simply downloading the tar.gz version hosted by github) and what they say in their docu.

Obviously, like @ejci  pointed,  the only possible thing to solve "partially" this issue is to move the examples to another branch.

I have seen some libraries (lo-dash comes to my mind) that have automated the process of creating builds and publishing them outside the development branch. In case of lo-dash they have modularized all their methods and have a full version that is generated by joining all of them. This allows them to generate custom builds https://lodash.com/custom-builds. Maybe a similar approach (even with git hooks could work well)

Recap of possible solutions:
a) move samples and unnecesary files to another branch 
b) script that generates builds ignoring the unnecessary files
